### DISCUSSION TITLE
Prevent networks from force-enabling Restricted Mode

### DISCUSSION THREAD

**Discussion by IndigoSamurott** [ReVanced/revanced-discussions#551] [Question]
If possible, a patch able to circumvent this:

![Screenshot_20220829-172154](https://user-images.githubusercontent.com/96881880/187248812-096c58ce-cd43-473c-b013-bc0ea040d004.png)

would be very useful and highly appreciated, as otherwise the use of a VPN is required - a pain in itself, but especially on (for example) already-slow hotel WiFi connections when we just want to be able to read comments.

**Comment by jamesbruice** [comment]
A workaround is using a VPN
